core/loadbalance/session_stickiness_strategy.go

```diff
@@ -34,24 +35,24 @@ func ResetSuccessiveFailureMap() {
 }
 
 //IncreaseSuccessiveFailureCount increase failure count
-func IncreaseSuccessiveFailureCount(ep string) {
+func IncreaseSuccessiveFailureCount(cookie string) {
```

Comment:
how about rename cookie -> cookieKey

Reply:
Sorry, cookieValue

Reply:
ok. I will change cookie to cookieValue.